ufactory_lite6/lite6_gripper_external.xml

```diff
@@ -0,0 +1,152 @@
+<mujoco model="ufactory_lite6">
+  <compiler angle="radian" meshdir="assets" autolimits="true"/>
+
+  <option integrator="implicitfast"/>
+
+  <default>
+    <default class="lite6">
+      <geom type="mesh"/>
+      <joint axis="0 0 1" damping="1" armature="0.1"/>
+      <general gaintype="fixed" biastype="affine" gainprm="2000" biasprm="0 -2000 -200"/>
```

Comment:
Do you want to switch to position actuators which now support damping? This can be converted to:

`<position kp="2000" kv="200"/>`

Reply:
FYI, if you update this, make sure you change the minimum required version for MuJoCo at the top of the README.

Reply:
Done. Updated Mujoco version to 3.1.0 when kv was introduced for position actuators as per [changelog](https://mujoco.readthedocs.io/en/latest/changelog.html#version-3-1-0-december-12-2023)